api/utils.py

```diff
@@ -144,6 +145,19 @@ def formatWorkOutput(
                 outWorks.append(outWork)
 
             return outWorks
+        #Formatted work with a specific format given
+        elif formats != None and identifiers == None:
+            formattedWork = cls.formatWork(
+                works, None, showAll, formats, reader=reader
```

Comment:
I think making this `formats=formats` is more consistent with how this is invoked above (though there is no reason it doesn't work!)

Reply:
Ok. I'll change the format parameter to match that then.